src/screens/Distribution/RingChart.js

```diff
@@ -0,0 +1,143 @@
+// @flow
+
+import React, { PureComponent } from "react";
+import { PanResponder, View } from "react-native";
+import * as d3shape from "d3-shape";
+import Svg, { Path, G, Circle } from "react-native-svg";
+import colors from "../../colors";
+import type { DistributionItem } from "./DistributionCard";
+
+type Props = {
+  data: Array<DistributionItem>,
+  side: number,
+  style?: *,
+  selectedKey: number => void,
```

Comment:
```suggestion
  onHighlightChange: number => void,
```

probably better naming

Reply:
ok